src/routes/auth.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  Box,
+  Button,
+  Container,
+  CssBaseline,
+  TextField,
+  ThemeProvider,
+  Typography,
+} from "@mui/material";
+import { darkTheme } from "../theme";
+
+export default function Auth() {
+  return (
+    <ThemeProvider theme={darkTheme}>
+      <Container component="main" maxWidth="xs" disableGutters>
+        <CssBaseline />
+        <Box
+          sx={{
+            marginTop: 8,
+            display: "flex",
+            flexDirection: "column",
+            alignItems: "center",
+          }}
+        >
+          <Typography variant="h2" align="center" gutterBottom color={"cyan"}>
+            Event Planner
+          </Typography>
+          <Box
+            sx={{
+              mt: 10,
+            }}
+          >
```

Comment:
Would rather use the "spacing" prop of the Stack, 
You can Stack a Stack inside a Stack how many times you want.